Make modperl_thx_interp_get() always return a value, fixing
undefined behaviour that led to a SIGBUS on sparc (#468327).
--- libapache2-mod-perl2.orig/src/modules/perl/modperl_interp.c
+++ libapache2-mod-perl2/src/modules/perl/modperl_interp.c
@@ -581,7 +581,7 @@
     modperl_interp_t *interp;
     dTHXa(thx);
     SV **svp = hv_fetch(PL_modglobal, MP_THX_INTERP_KEY, strlen(MP_THX_INTERP_KEY), 0);
-    if (!svp) return;
+    if (!svp) return NULL;
     interp = INT2PTR(modperl_interp_t *, SvIV(*svp));
     return interp;
 }
